### PR TITLE
[FEATURE] - Resource Count Delection Skipping

### PR DESCRIPTION
**Pull request by gambol99**
When then configuration has not provisioned any ressources, we can skip the deletion job

### CODE REVIEW

**Comment by gambol99** [comment]
Related to https://github.com/appvia/terranetes-controller/issues/1187

File: pkg/controller/configuration/delete.go [hunk 1]
```diff
@@ -11,7 +11,7 @@
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU General Public License
+ * You should have received a copy of thRe GNU General Public License
```
Comment (KashifSaadat) [inline pkg/controller/configuration/delete.go:14]: typo

File: pkg/controller/configuration/delete_test.go [hunk 2]
```diff
@@ -132,6 +133,55 @@ var _ = Describe("Configuration Controller with Contexts", func() {
...
+			})
+
+			Context("but the configuration has no resources", func() {
+				BeforeEach(func() {
+					configuration.Status.Resources = ptr.To(0)
+
+					result, _, rerr = controllertests.Roll(context.TODO(), ctrl, configuration, 0)
+				})
+
+				It("should not create thhe destroy job", func() {
```
Comment (KashifSaadat) [inline pkg/controller/configuration/delete_test.go:177]: typo

Comment (KashifSaadat) [reply pkg/controller/configuration/delete_test.go:177]: can you assert on the expected event/status (`DeletionSkipped`)?

**Comment by gambol99** [comment]
Fixed up as suggested in https://github.com/appvia/terranetes-controller/pull/1194/commits/fb63ce685e85c62a3f5adaef20f8e8b970704ada @KashifSaadat

**Review by KashifSaadat — APPROVED** [review]
LGTM, thanks!!

**Pull request merged**